json_serializable/test/integration/create_per_field_to_json_example.g.dart

```diff
@@ -0,0 +1,127 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
+
+// ignore_for_file: lines_longer_than_80_chars, text_direction_code_point_in_literal
+
+part of 'create_per_field_to_json_example.dart';
+
+// **************************************************************************
+// JsonSerializableGenerator
+// **************************************************************************
+
+Model _$ModelFromJson(Map<String, dynamic> json) => Model(
+      firstName: json['firstName'] as String,
+      lastName: json['lastName'] as String,
+      enumValue: $enumDecodeNullable(_$EnumValueEnumMap, json['enumValue']),
+      nested: json['nested'] == null
+          ? null
+          : Nested.fromJson(json['nested'] as Map<String, dynamic>),
+      nestedExcludeIfNull: json['nestedExcludeIfNull'] == null
+          ? null
+          : Nested.fromJson(
+              json['nestedExcludeIfNull'] as Map<String, dynamic>),
+      nestedGeneric: json['nestedGeneric'] == null
+          ? null
+          : GenericFactory<int>.fromJson(
+              json['nestedGeneric'] as Map<String, dynamic>,
+              (value) => value as int),
+    );
+
+// ignore: unused_element
+abstract class _$ModelPerFieldToJson {
+// ignore: unused_element
```

Comment:
Seems like the formatter doesn't help with these. In that case, it might be good if YOU indent them in the output